src/clusterfuzz/_internal/base/tasks.py

```diff
@@ -188,9 +228,13 @@ def get_regular_task(queue=None):
   application_id = utils.get_application_id()
   while True:
```

Comment:
nit: not really relevant for this PR, but can we add max_messages=1 to line 230 here to be more explicit about the expected behaviour?